### PR TITLE
AP_HAL_ChibiOS: default param updates for KakuteH7 autopilots

### PR DESCRIPTION
**Pull request by Hwurzburg**
to match BF setups better

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/KakuteH7v2/defaults.parm [hunk 1]
```diff
@@ -3,4 +3,9 @@ SERVO9_FUNCTION 120
 NTF_LED_TYPES 257
 # Default VTX power to on
 RELAY_DEFAULT 1
-SERIAL7_PROTOCOL 16
\ No newline at end of file
+# UART1 for DJI Goggles
```
Comment (andyp1per) [inline libraries/AP_HAL_ChibiOS/hwdef/KakuteH7v2/defaults.parm:6]: You were going to add GPS?

**Review by andyp1per — CHANGES_REQUESTED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/KakuteH7v2/defaults.parm (hunk 1 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/KakuteH7v2/defaults.parm:6]: automatically assigned to 3 and 4

**Comment by Hwurzburg** [comment]
@andyp1per fixed mini v1.3 also

File: libraries/AP_HAL_ChibiOS/hwdef/KakuteH7Mini-Nand/defaults.parm [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# setup for LEDs on chan9
+SERVO9_FUNCTION 120
+NTF_LED_TYPES 257
+# Default VTX power to on
+RELAY_DEFAULT 1
+# UART1 for DJI Goggles
+SERIAL1_PROTOCOL 33
```
Comment (andyp1per) [inline libraries/AP_HAL_ChibiOS/hwdef/KakuteH7Mini-Nand/defaults.parm:7]: Good catch. Can you add the DJI protocol to the regular H7 Mini?

File: libraries/AP_HAL_ChibiOS/hwdef/KakuteH7v2/defaults.parm [hunk 3]
```diff
@@ -3,4 +3,9 @@ SERVO9_FUNCTION 120
...
+SERIAL1_PROTOCOL 33
+# UART3 for VTX
+SERIAL3_PROTOCOL 37
+#ESC telem on RX7
+SERIAL7_PROTOCOL 16
```
Comment (andyp1per) [inline libraries/AP_HAL_ChibiOS/hwdef/KakuteH7v2/defaults.parm:11]: This is actually unnecessary - already defined in hwdef.dat

**Review by andyp1per — CHANGES_REQUESTED** [review]
Can you fix the two comments? Looks good!

File: libraries/AP_HAL_ChibiOS/hwdef/KakuteH7v2/defaults.parm (hunk 3 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/KakuteH7v2/defaults.parm:11]: did not add that...but can remove...

File: libraries/AP_HAL_ChibiOS/hwdef/KakuteH7Mini-Nand/defaults.parm (hunk 2 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/KakuteH7Mini-Nand/defaults.parm:7]: done

File: libraries/AP_HAL_ChibiOS/hwdef/KakuteH7v2/defaults.parm (hunk 3 above)
Comment (andyp1per) [reply libraries/AP_HAL_ChibiOS/hwdef/KakuteH7v2/defaults.parm]: Yeah, realised - just seemed worth fixing while you are in here

**Review by andyp1per — APPROVED** [review]

**Pull request merged**